doc/ale.txt

```diff
@@ -1990,6 +1990,24 @@ g:ale_windows_node_executable_path         *g:ale_windows_node_executable_path*
   scripts are executed with whatever executable is configured with this
   setting.
 
+g:ale_shell                                                       *g:ale_shell*
+
+  Type: |String|
+  Default: `'&shell'`
+
+  This variable is used to determine which shell ale will use to execute 
+  commands. This variables defaults to the value of the vim option '&shell'
+  which corresponds to the $SHELL environment variable. For example 
+  if `$SHELL == '/bin/bash'`, but you want to use zsh, set `g:ale_shell = '/bin/zsh'.`
```

Comment:
add in a comment to suggest using `g:ale_shell_arguments` if overriding this, as `g:ale_shell_arguments` can pull incompatible flags from `&shellcmdflag` even if the user changes shell